DOCSP-31483: tls cert files read async v6 (#736)
mongodb/docs-node: source/fundamentals/connection/connection-options.txt

     - ``false``
     - Specifies whether TLS is required for connections to the server.
       Using a ``srvServiceName`` of  ``"mongodb+srv"``, or specifying other

>        ``tls`` prefixed options will default ``tls`` to ``true``. To

Reviewer: Issues:

1) Word choice:
I think "default" used as a verb is usually associated with failure to take an action or pay. 

2) Use present tense:
Also, I think this could use [present tense](https://www.mongodb.com/docs/meta/style-guide/writing/use-present-tense/) instead of future tense.

Suggestion:

```suggestion
       ``tls`` prefixed options sets the value of ``tls`` to ``true``. To
```
